### PR TITLE
minor fixes to get server working, which I think were things which di…

### PR DESCRIPTION
**Pull request by geoffjentry**
…sappeared in a merge anyways

### CODE REVIEW

**Comment by geoffjentry** [comment]
@scottfrazer @kshakir

**Comment by geoffjentry** [comment]
Oops, I assigned two people.

@scottfrazer

**Comment by scottfrazer** [comment]
Reviewer: @mcovarr

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -1,6 +1,8 @@
-webservice.port = 8000
-webservice.interface = 0.0.0.0
-instance.name = "reference"
+webservice {
+  port = 8000
+  interface = 0.0.0.0
+  instance.name = "reference"
+}
```
Comment (mcovarr) [inline src/main/resources/application.conf]: yay

File: src/main/scala/cromwell/server/CromwellServer.scala [hunk 2]
```diff
@@ -47,7 +47,7 @@ object CromwellServer extends DefaultWorkflowManagerSystem {
   val webserviceConf = conf.getConfig("webservice")
 
   import scala.concurrent.ExecutionContext.Implicits.global
-  (IO(Http) ? Http.Bind(service, interface =  webserviceConf.getString("webservice.interface"), port = webserviceConf.getInt("webservice.port"))).onComplete {
+  (IO(Http) ? Http.Bind(service, interface =  webserviceConf.getString("interface"), port = webserviceConf.getInt("port"))).onComplete {
```
Comment (mcovarr) [inline src/main/scala/cromwell/server/CromwellServer.scala]: could `.ask` and fix the messed up spacing while you're in here.  :stuck_out_tongue:

File: src/main/resources/application.conf [hunk 3]
```diff
@@ -1,6 +1,8 @@
-webservice.port = 8000
-webservice.interface = 0.0.0.0
-instance.name = "reference"
+webservice {
+  port = 8000
+  interface = 0.0.0.0
```
Comment (mcovarr) [inline src/main/resources/application.conf]: this string doesn't need quotes around it?

Comment (scottfrazer) [inline src/main/resources/application.conf]: This is the only question I have, besides that looks fine to me

Comment (geoffjentry) [inline src/main/resources/application.conf]: Sorry, spoke in person to @mcovarr - it's just the magic of hocon

**Pull request merged**